src/repos/messageRepo.js

```diff
@@ -0,0 +1,35 @@
+const BaseRepo = require('./baseRepo')
+const { Message } = require('../entities')
+
+class MessageRepo extends BaseRepo {
+    static get entity() {
+        return Message
+    }
+
+    static get table() {
+        return 'messages'
+    }
+
+    async create(message) {
+        const [record] = await this.query
+            .insert({
+                room_id: message.attr.roomId,
```

Comment:
Do I need a getter for each of the attributes?
What happens if it's a big entity? 5 getters is not much, but what if the entity had 20 attributes, would I need to create 20 getters?
But generally what difference does it make? I mean getting the attribute either through a getter or `attr`?

Reply:
Because if need to to customize logic of getter it will be better to do in one place. Also, you should not have entity with may getter (and tables with many columns). This is kinda protection :) 
Also you can have composite attribute with object type. (group many attributes in one)( settings for instance)

Reply:
So let's sey if I had an entity that has 20+ attributes, I'd need to break it down into smaller entities?

Reply:
So let's sey if I had an entity that has 20+ attributes, I'd need to break it down into smaller entities?

Reply:
This is one possible solution, yes

Reply:
20+ is not many.